src/main/java/org/schemaspy/connection/Filed.java

```diff
@@ -0,0 +1,35 @@
+package org.schemaspy.connection;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Properties from this file (in key=value pair format) are passed to the
+ * database connection.<br>
+ * user (from -u) and password (from -p) will be passed in the
+ * connection properties if specified.
+ */
+public class Filed {
+
+    private final String propertiesFilename;
+
+    /**
+     * @param propertiesFilename file to use for connection properties
+     */
+    public Filed(String propertiesFilename) {
+        this.propertiesFilename = propertiesFilename;
+    }
+
+    /**
+     * @throws IOException if we have problems reading the file
+     */
+    public Properties setConnectionPropertiesFile() throws IOException {
```

Comment:
set seems awkward when constructing a new object and not changing any state.

Reply:
Well the previous methods were setting a state, now they are not. But they retained the "set" prefix.

Which is confusing, like it's mid refactoring. I changed the method body but not the name.